packages/radix-ui-themes/src/components/data-list.css

```diff
@@ -0,0 +1,98 @@
+/* Root -------------------------------*/
+.rt-DataListRoot {
+  margin: 0;
+  display: grid;
+  /** Value  set via props in component **/
+  grid-template-columns: auto 1fr;
+}
+
+/* Item -------------------------------*/
+.rt-DataListItem {
+  position: relative;
+  box-sizing: border-box;
+  grid-column: span 2;
+}
+
+@breakpoints {
+  /* Orientation */
```

Comment:
These however I would keep and actually use big comment blocks like we have already for SIZES/VARIANTS, etc

This way I would group by prop, rather than group by part for those styles.
If you look at other components, you'll see what I mean, basically you'd find all the code necessary for a specific prop under the same block.